### ISSUE TITLE
Sample App Documentation Rendering Issues

### ISSUE THREAD

**Issue by michael-hawker** [CommunityToolkit/Windows#215]
Known Issues:
- Markdown Table in Attached Shadow Overview doesn't render
- DocFX call outs don't render nice

Hoping we can just swap to the new control from Labs - https://github.com/CommunityToolkit/Labs-Windows/pull/480
And then just add any missing features for rendering on top of that instead.

**Comment by michael-hawker** [comment]
Also `CameraPreview` sample doesn't stop using camera when navigating away. Not sure if that's a sample page issue or a problem with the control, but we'll have to fix it later. Probably its own bug? Considering it doesn't work due to Windows App SDK bug, it's probably not a high priority? But still going to be an issue.

**Comment by itsWindows11** [comment]
Also it seems to take a few secs to load any sample?

**Comment by Arlodotexe** [comment]
> Are warnings/notes intended to be displayed like this?

No, these are intended to be rendered properly. Tracked in https://github.com/CommunityToolkit/Tooling-Windows-Submodule/issues/70
